src/components/Auth/SignUpForm.js

```diff
@@ -4,26 +4,22 @@ import { Link, Redirect } from 'react-router-dom';
 import { Box, Button, TextField } from '@material-ui/core/';
 import axios from 'axios';
 import { useAuth } from './AuthContext';
+import { validationResolver } from './SignUpForm.schema';
+import Form, { Field } from '../form';
 
 const SignUpForm = ({ toggleActiveForm }) => {
-  const [firstName, setFirstName] = useState(null);
-  const [lastName, setLastName] = useState(null);
-  const [username, setUsername] = useState(null);
-  const [email, setEmail] = useState(null);
-  const [password, setPassword] = useState(null);
   const [errorMessage, setErrorMessage] = useState(null);
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const referer = '/profile';
   const auth = useAuth();
 
-  const handleSignup = e => {
-    e.preventDefault();
+  const onSubmit = formData => {
```

Comment:
I wonder if the changes here are related to this error when I'm typing into a form input:
![image](https://user-images.githubusercontent.com/4512699/80044967-f0629f00-84ba-11ea-8e34-74591595f27c.png)

Reply:
Yeah, actually this is related to setting the default value to the fields, Will check...

Reply:
Fixed!